.github/workflows/docker-build.yaml

```
@@ -0,0 +1,63 @@
+
+    - name: Run tests
+      if: success()  # Only run if pre-commit passes
+      run: |
+        pytest tests/
+
+    - name: Set up Cloud SDK
+      if: success()
+      uses: google-github-actions/setup-gcloud@v2
+      with:
+        project_id: ${{ env.PROJECT_ID }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
```

actionlint (1.7.4)

input "service_account_key" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)